chart/templates/uds-package-sandbox.yaml

```
@@ -0,0 +1,47 @@
+apiVersion: uds.dev/v1alpha1
+kind: Package
+metadata:
+  name: gitlab-runner-sandbox
+  namespace: {{ .Values.runnerNamespace }}
```

```suggestion
  namespace: {{ .Values.sandboxNamespace }}
```
This may be more clear (yes breaking but would align more with `customSandbox`)

---

Can of worms because now its like... lets remove that default svc account from both namespaces as well.